src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt

```diff
@@ -157,4 +159,9 @@ class PaymentRequestsService(
   }
 
   fun getDueDateString(date: XMLGregorianCalendar?): String? = date?.toString()
+
+  private fun dueDateToLocalDate(date: XMLGregorianCalendar): String =
+    ZonedDateTime.ofInstant(date.toGregorianCalendar().toInstant(), ZoneId.of("Europe/Paris"))
+      .toLocalDate()
+      .toString()
```

Comment:
this can be simplified as follow: 
```suggestion
    LocalDate.ofInstant(date.toGregorianCalendar().toInstant(), ZoneId.of("Europe/Paris"))
      .toString()
```

Reply:
Changed in 34decc6